SouvlakMVP/SouvlakMVP/VerticesConnections.cs

```diff
@@ -74,6 +96,10 @@ public void Rebuild(Graph graph)
         }
     }
 
+    /// <summary>
+    /// Initializes a new instance of VerticesConnections class
+    /// </summary>
+    /// <param name="graph">Graph from which connection matrix will be created</param>
     public VerticesConnections(Graph graph)     // Yes I know of DRY, but VS2022 kept giving me warnings if I didn't do it like this
```

Comment:
DRY -> hmm...
wouldn't it work to use constructor `VerticesConnections()` in `Rebuild()`?
I see that they are the same:
https://github.com/Kri5t0fK/SouvlakiSearch/blob/a7131cca7fe2abab010fea736867ea6c1d2b9661/SouvlakMVP/SouvlakMVP/VerticesConnections.cs#L85-L97 https://github.com/Kri5t0fK/SouvlakiSearch/blob/a7131cca7fe2abab010fea736867ea6c1d2b9661/SouvlakMVP/SouvlakMVP/VerticesConnections.cs#L103-L115

Reply:
If you don't mind warnings then it would. But VS is not as smart as it seems to be :/

Reply:
To elaborate more: you can't call constructor inside the same class in C#. You theoretically could call Rebuold() inside constructor, but VS doesn't recognise, that Rebuild() initizalizes all fields and gives an ugly warning. And I HATE warnings :3